domains/certificates/Query.API/API.AppTests/ApiTests.GetList_FiveMeasurementAddedToBus_ReturnsList.verified.txt

```diff
@@ -0,0 +1,54 @@
+﻿{
+  Result: [
+    {
+      Id: Guid_1,
+      DateFrom: 1654056000,
+      DateTo: 1654059600,
+      Quantity: 46,
+      GSRN: Guid_2,
+      GridArea: GridArea,
+      TechCode: bar,
+      FuelCode: foo
+    },
+    {
+      Id: Guid_3,
```

Comment:
Where is guid 2?

Reply:
Verify.Xunit is scrubbing the data. So here it says that it expects a Guid and it labels that as "Guid_2", but it does not care about the specific value. In the file "Guid_2" is used for all GSRN properties, which means that it must be the same value.

It is nice in our case because the Id property is auto-generated by the code. And GSRN is auto-generated by the test.

Reply:
But looking at it now, I think it would make sense to use a specific name for GSRN, like "123456" or "gsrn-1" since GSRN is not actually a Guid.

Reply:
Changed in 7a01adc9e88d2871ca2647c4364ca8a1dd56fffd